boot/zephyr/Kconfig

```diff
@@ -663,10 +663,6 @@ config MULTITHREADING
 	default n if SOC_FAMILY_NRF
 	default y
 
-config LOG_IMMEDIATE
```

Comment:
There is one more occurrence of `LOG_IMMEDIATE` in this file on line 445, however I believe it's usage is consistent with how the symbol is supposed to be used / is used by Zephyr. Please double check.

Reply:
Yes, it is consistent.